src/main/java/com/ait/lienzo/client/core/shape/wires/SelectionManager.java

```diff
@@ -844,6 +843,17 @@ public void clear()
             m_selManager.m_startBoundingBox = null;
         }
 
+        public void destroy()
+        {
+            selectionGroup  = false;
+            m_changed.clear();
+            m_shapes.clear();
+            m_connectors.clear();
+            m_externallyConnected.clear();
+            m_bbox = null;
+            m_selManager.m_startBoundingBox = null;
```

Comment:
Similar question, looking on list of class memebers, there seems to me more complex objects that should be destroyed somehow? But sure, I know basically nothing about lienzo, so takes this just as question.